code/__DEFINES/subsystems.dm

```diff
@@ -149,4 +150,6 @@
 			}\
 		}\
 		A.flags_1 &= ~OVERLAY_QUEUED_1;\
+		if(isturf(A)){SSdemo.mark_turf(A);}\
+		if(isobj(A) || ismob(A)){SSdemo.mark_dirty(A);}\
```

Comment:
How much performance have we lost because this isn't an `else if` 😆

Reply:
none you fucking spoon, stop wasting time on meaningless microops that get lost in the noise

Reply:
It gets called a LOT tbf